### ISSUE TITLE
Send out ScienceMesh invite over email (if you fill in the email address)

### ISSUE THREAD

**Issue by mrvahedi68** [pondersource/sciencemesh-nextcloud#204]

**Comment by parhamin2010** [comment]
@mrvahedi68 could you please clarify this?
The thing I understand is this:
- Create a new button that input the user email
- After generate token it will sent it by getting the value of that input and automatically send it to mentiond email.

**Comment by mrvahedi68** [comment]
Yes that's true, but I see a line inside the reva source code that seems to sends the invitation link to recipient email. I think we need only add an option to set the recipient email.
Reva code for sending email: https://github.com/cs3org/reva/blob/c6072ab50f90818a3555d0069c3ca7fce3433e25/internal/http/services/sciencemesh/token.go#L113

**Comment by parhamin2010** [comment]
@mrvahedi68 
Please review the Reva side to see if it's sending properly or not. I've not recieved anything once I tried to send by set recipient value to endpoint.
